apps/forum/vars/vault/main.yml.j2

```diff
@@ -6,7 +6,7 @@
 {% set mongodb_credentials = databases.mongodb | json_query("[?release=='" ~ forum_mongodb_version ~ "'].databases | [0][?application=='forum'].{user: user, password: password, name: name} | [0]") %}
 MONGODB_ADMIN_PASSWORD: password
 MONGODB_USER: {{ mongodb_credentials.user }}
-MONGODB_PASSWORD: {{ mongodb_credentials.password }}
+MONGODB_PASSWORD: {{ mongodb_credentials.password | urlencode }}
```

Comment:
This will break everything, you should instead urlencode the password on-the-fly when defining the `MONGOHQ_URL` environment variable in the forum's DC and Job.

Reply:
:+1: